presto-docs/src/main/sphinx/release/release-0.244.rst

```diff
@@ -0,0 +1,48 @@
+=============
+Release 0.244
+=============
+
+**Highlights**
+==============
+* Improve performance of NestedLoopJoinOperator.
+* Add support for spilling joins when using materialized exchanges.
+
+
+**Details**
+===========
+
+General Changes
+_______________
+* Fix an issue where queries could encounter an int overflow error during spilling.
+* Fix hamming_distance to handle code point zero correctly.
+* Fix caching and case-sensitivity bugs with type signatures for enum types.
+* Fix LambdaDefinitionExpression canonicalization to correctly canonicalize Block constant (:issue:15424).
+* Improve performance of NestedLoopJoinOperator.
+* Add `peak_node_total_memory` to statistics to allow monitoring the peak sum of memory used across tasks at any node by a query.
+* Add support for spilling joins when using materialized exchanges.
+* Add thrift transport support for TaskStatus which can be enabled using config property ``experimental.internal-communication.thrift-transport-enabled``.
+
+Verifier Changes
+________________
+* Fix an issue where queries are always directed to the first cluster when multiple control or test clusters are specified.
+* Improve verifier handling of memory limits by using the limits from the verifier cluster which are frequently better matched to the queries picked to be run in the verifier suite.
+* Add control and test session properties to Verifier output.
+
+Druid Changes
+_____________
+* Fix druid reading issue.
```

Comment:
You can check the original PR to see if we can expand the description here, in terms of what is the issue.